home/widgets.py

```diff
@@ -59,28 +61,73 @@ def _observe_message(self, change):
         self.show_temporary_message(change['new'])
 
 
-class UpdateAvailableInfoWidget(ipw.HTML):
+class AppStatusInfoWidget(ipw.HTML):
     """Widget that indicates whether an update is available."""
```

Comment:
Maybe change the docstring to better reflect the changes.

Reply:
Done.